jnigen/examples/pdfbox_plugin/lib/third_party/_init.dart

```diff
@@ -0,0 +1,23 @@
+// Generated from Apache PDFBox library which is licensed under the Apache License 2.0.
+// The following copyright from the original authors applies.
+//
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import "dart:ffi";
+import "package:jni/internal_helpers_for_jnigen.dart";
+
+final Pointer<T> Function<T extends NativeType>(String sym) jlookup =
```

Comment:
`jlookup` -> `lookup`/`javaLookup`?

Reply:
lookup is ruled out, because it might conflict with some variable.

`javaLookup` maybe.